### PR TITLE
Update to version 0.9.0

### PR DESCRIPTION
**Pull request by poblahblahblah**

### CODE REVIEW

**Comment by calvn** [comment]
Consul releases now excludes the web_ui as a separate bundle. We should remove all reference to it (download.bat and chocolateyInstall.ps1).

**Comment by poblahblahblah** [comment]
@calvn fixed.

File: tools/chocolateyInstall.ps1 [hunk 1]
```diff
@@ -74,7 +69,7 @@ if ($service) {
 
 Write-Host "Installing service: $serviceName"
 # Install the service
-& $wrapperExe install $serviceName $(Join-Path $toolsPath "consul.exe") "agent -ui-dir=$serviceUiDirectory -config-dir=$serviceConfigDirectory -data-dir=$serviceDataDirectory $packageParameters" | Out-Null
+& $wrapperExe install $serviceName $(Join-Path $toolsPath "consul.exe") "agent -ui -config-dir=$serviceConfigDirectory -data-dir=$serviceDataDirectory $packageParameters" | Out-Null
```
Comment (calvn) [inline tools/chocolateyInstall.ps1:72]: We can also remove these lines: 

https://github.com/calvn/chocolatey-consul/blob/master/tools/chocolateyInstall.ps1#L9
https://github.com/calvn/chocolatey-consul/blob/master/tools/chocolateyInstall.ps1#L25
https://github.com/calvn/chocolatey-consul/blob/master/tools/chocolateyInstall.ps1#L37

Comment (poblahblahblah) [reply tools/chocolateyInstall.ps1:72]: @calvn those lines are already removed in my PR

Comment (calvn) [reply tools/chocolateyInstall.ps1:72]: My bad, I didn't realize!

Comment (poblahblahblah) [reply tools/chocolateyInstall.ps1:72]: No worries! Thank you!

**Pull request merged**